Examples/PcapPlusPlus-benchmark/benchmark-google.cpp

```diff
@@ -142,6 +142,51 @@ static void BM_PacketParsing(benchmark::State& state)
 }
 BENCHMARK(BM_PacketParsing);
 
+static void BM_PacketPureParsing(benchmark::State& state)
+{
+	pcpp::PcapFileReaderDevice reader(pcapFileName);
+	if (!reader.open())
+	{
+		state.SkipWithError("Cannot open pcap file for reading");
+		return;
+	}
+
+	// Preloads all packets into memory
+	pcpp::RawPacketVector rawPackets;
+	reader.getNextPackets(rawPackets);
+
+	if (rawPackets.size() == 0)
+	{
+		state.SkipWithError("No packets to parse");
+		return;
+	}
+
+	size_t totalProcessedItems = 0;
+	size_t totalProcessedBytes = 0;
+	size_t currentPacketIndex = 0;
+	for (auto _ : state)
+	{
+		pcpp::RawPacket* rawPacket = rawPackets.at(currentPacketIndex);
+
+		pcpp::Packet parsedPacket(rawPacket);
+
+		benchmark::DoNotOptimize(parsedPacket.getFirstLayer());
+
+		++totalProcessedItems;
+		++totalProcessedBytes += rawPacket->getRawDataLen();
```

Comment:
Fix wrong total bytes increment.

```suggestion
		totalProcessedBytes += rawPacket->getRawDataLen();
```